release_notes/45-release-notes.adoc

```diff
@@ -0,0 +1,421 @@
+
+* {product-title-short} includes new policy categories, and some default policies are tagged with these new policy categories.
++
+.New policy categories
+[options="header"]
+|====
+
+|Policy category|Policy
+
+.2+|*Zero Trust*
+|Deployments should have at least one ingress Network Policy
+|Unauthorized Network Flow
+
+.7+|*Supply Chain Security*
+|Images with no scans
+|30-day Scan Age
+|90-day Image Age
+|Required Annotation: Email
+|Required Annotation: Owner/Team
+|Required Label: Owner/Team
+|Latest tag
+
+|====
+
+* When generating manifests for {ocp}, `roxctl` now defaults to {ocp} 4.x instead of {ocp} 3.x.
+
+* You can now delegate the Image scans triggered by image watch reprocessing based on the delegated scanning configuration. To turn off this feature, set the Central environment variable `ROX_DELEGATE_WATCHED_IMAGE_REPROCESSING` to `false`.
+
+* The Scanner V4 Matcher now performs concurrent vulnerability updates using iterators, reducing memory consumption from 4GB to 500MB.
+
+* {product-title-short} 4.5 lazily populates the initial registry integration repository list `/v2/_catalog` to improve Central startup performance. This change reduces startup time in environments with many autogenerated integrations.
+
+* {product-title-short} 4.5 includes a new configuration option, `ROX_SCANNER_V4_ALLOW_ANONYMOUS_AUTH`, to enable anonymous access to Scanner V4 for debugging purposes. This feature is on by default for development builds and off for release builds.
+
+* Deployment bundles created with the roxctl CLI no longer contain PodSecurityPolicies (PSPs) by default. You must specify the `--enable-pod-security-policies` option when generating deployment bundles for deploying to Kubernetes 1.25 or earlier.
+
+* {product-title-short} 4.5 includes improved event handling for Sensor image scan when `ROX_UNQUALIFIED_SEARCH_REGISTRIES` is set to `true`. This enhancement ensures that only one simultaneous scan request is allowed per unique image, reducing redundant scans. Additionally, it increases the chances of scan cache hits when multiple names for the same image are observed. This feature is on by default when `ROX_UNQUALIFIED_SEARCH_REGISTRIES` is `true`; to turn it off, set  `ROX_SENSOR_SINGLE_SCAN` to `false` on Sensor.
+
+* Red{nbsp}Hat has reduced the default timeout setting for {product-title-short} admission controller webhooks from 20 seconds to 10 seconds, resulting in an effective timeout of 12 seconds within the ValidatingWebhookConfiguration. This change aligns with {ocp}'s unconditional cap of 13 seconds. If you rely on longer timeouts, such as those with inline image scanning, you must specify a longer timeout explicitly in the `SecuredCluster` custom resource `admissionControl.timeoutSeconds`, in Helm `admissionControl.dynamic.timeout`, or within a sensor deployment bundle `ValidatingWebhookConfiguration` manifest within the `admission-controller.yaml` file.
```

Comment:
```suggestion
* Red{nbsp}Hat has reduced the default timeout setting for {product-title-short} admission controller webhooks from 20 seconds to 10 seconds, resulting in an effective timeout of 12 seconds within the ValidatingWebhookConfiguration. This change aligns with the unconditional cap of 13 seconds in {ocp}. If you rely on longer timeouts, such as those with inline image scanning, you must specify a longer timeout explicitly in the `SecuredCluster` custom resource `admissionControl.timeoutSeconds`, in Helm `admissionControl.dynamic.timeout`, or within a sensor deployment bundle `ValidatingWebhookConfiguration` manifest within the `admission-controller.yaml` file.
```
Do not use ’s to show the possessive form of abbreviations or product names: https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives